README.md

```diff
@@ -22,6 +22,7 @@ To build your own typechecker:
   Central. Change the name of the package containing the source (currently
   `org.checkerframework.checker.templatefora`) so that it reflects your
   organization's naming standards.
+* Replace all instances of `org.checkerframework.checker.templatefora` and `org/checkerframework/checker/templatefora`. For example, change the path `**/org/checkerframework/checker/templatefora/qual/**` specified in the `templatefora-checker-qual/build.gradle` with your actual path.
```

Comment:
This bullet point should be combined with the one above, which already instructs the user to rename the package. I would not specifically call out the location that needs to be changed in `templatefora-checker-qual/build.gradle`, and instead add only the first sentence of your change to the end of the previous bullet.

Reply:
@kelloggm 
I have made an update according to your last comment.